force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -589,6 +612,13 @@
         <shortDescription>Total Sessions</shortDescription>
         <value>Total Sessions</value>
     </labels>
+    <labels>
+        <fullName>User</fullName>
```

Comment:
@bethbrains - what are your thoughts on using alternative-text on the lightning-icon for user, my gut says since it is an icon for purely decorative purposes and not informational we wouldn't need it?